src/adding_datasets_to_kg/convert_data.py

```diff
@@ -56,9 +56,42 @@ def convert_cbioportal_data():
                                        predicate="biolink:gene_associated_with_condition",
                                        object_id=disease_id,
                                        primary_knowledge_source="infores:tcga")
+            
+def convert_1kg_data() -> None: 
+    print("Converting 1kg data to KGX files...")
+    onekg_data_path = get_data_directory_path() / "1kg" / "1kg_test2.json"
+    with (open(onekg_data_path, "r") as onekg_data_file,
+          get_kgx_output_file_writer("1kg") as kgx_file_writer):
+        for line in onekg_data_file:
+            variant_obj = json.loads(line)
+            variant_id = next((format_hgvsg(tc["hgvsg"], tc["spdi"]) for tc in variant_obj['transcript_consequences'] if "hgvsg" in tc), None)
+
+            if variant_id:
+                print(variant_id)
+                frequency_list = variant_obj["input"].split()[-1].split(";")
+                most_severe_consequence = f"UMLS:{variant_obj["most_severe_consequence"]}"
+                for frequency in frequency_list:
+                    if frequency.startswith("AFR="):
+                        AFR_frequency = {"AFR": frequency.split("=")[1]}
```

Comment:
Each of these population sets (AFR, AMR, EAS,EUR,SAS) can also be nodes and variants can connect to this with https://biolink.github.io/biolink-model/VariantToPopulationAssociation/  perhaps with frequency as edge property?